src/views/Timesheets/index.jsx

```diff
@@ -501,6 +501,24 @@ const Timesheets = (props) => {
                       </Link>
                     </Typography>
                   </Grid>
+                  <Grid item xs={12}>
+                    <Typography>
+                      <Link
+                        className="disable_select"
+                        style={{
+                          borderBottom: '1px solid currentColor',
+                          textDecoration: 'none',
+                          color: gordonColors.primary.blueShades.A700,
+                        }}
+                        href="https://www.timeanddate.com/worldclock/"
+                        underline="always"
+                        target="_blank"
+                        rel="noopener"
+                      >
+                        View time
+                      </Link>
+                    </Typography>
+                  </Grid>
```

Comment:
This seems unrelated to the changes that you are making with mailboxes. It's best to try and stay within scope for one PR.